src/App.js

```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+import Home from './Prensentationals/Home/Home';
+import LayoutDesign from './Prensentationals/Layout-Design/LayoutDesign';
```

Comment:
Careful with typos here - this can get you into a world of hurt with your imports. Check out the vsCode extension that Greg posted in Slack: Code Spell Checker. :)